### PR TITLE
Add CI step to build image-transformer

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:
In the PR #525 and #524 , using the image-transformer which is in the personal repo, we should build that and push to `gcr.io/kubeflow-ci/kfserving/image-transformer`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/526)

### CODE REVIEW

**Comment by pugangxa** [comment]
Thanks Jinchi for the quick response!!

File: test/scripts/build-image-transformer.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (animeshsingh) [inline test/scripts/build-image-transformer.sh]: This is a custom transformer image - I wouldn't give it such a generic name

File: test/workflows/components/workflows.libsonnet [hunk 2]
```diff
@@ -234,6 +234,10 @@
                     name: "build-kfserving-executor",
                     template: "build-kfserving-executor",
                   },
+                  {
+                    name: "build-image-transformer",
+                    template: "build-image-transformer",
+                  },
```
Comment (animeshsingh) [inline test/workflows/components/workflows.libsonnet]: Same here - a custom user transformer image if i am not mistaken

File: test/scripts/build-image-transformer.sh (hunk 1 above)
Comment (jinchihe) [reply test/scripts/build-image-transformer.sh]: Updated, thanks.

**Pull request merged**